src/main/java/org/reactome/release/dataexport/configuration/ConfigurationManager.java

```diff
@@ -1,5 +1,7 @@
 package org.reactome.release.dataexport.configuration;
 
+import static java.nio.file.attribute.PosixFilePermission.*;
```

Comment:
Could you explain how you use this package? It makes me think of an error we've had with certain steps in the release, either `PATHX` or `POSIX` (recalling from memory)

Reply:
It's a core Java package. I believe it is used for setting file permissions. I don't think we normally use this (or any other POSIX-related package). Can you explain the context of this error that you remember? Was it perhaps a certificate error? or maybe an "XPath" error?

Reply:
Yes, it was a certificate error!

Reply:
The code here is for setting permissions on local files. Should not have any connection at all to the certificate error you saw,.

Reply:
POSIX is a set of IEEE standards: https://en.wikipedia.org/wiki/POSIX